src/actionCreators/resources.js

```diff
@@ -173,7 +174,8 @@ export const expandProperty = (propertyKey, errorKey) => (dispatch, getState) =>
           return dispatch(addValueAction(newValue))
         })))
   } else {
-    property.values = []
+    property.values = defaultValuesFor(property)
```

Comment:
the bug was here: when expanding a property, make sure you add the defaults if they exist